api_tests/registrations/views/test_registration_detail.py

```diff
@@ -442,6 +454,109 @@ def test_read_write_contributor_cannot_update_custom_citation(
         assert res.status_code == 403
 
 
+@pytest.mark.django_db
+class TestRegistrationWithdrawal(TestRegistrationUpdateTestCase):
```

Comment:
Please help me dear reviewer, have I spelled and used `withdrawal` and `withdrawn` correctly?!